addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComInterfaceControlMessage.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.rfxcom.internal.messages;
+
+import java.util.List;
+
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.Type;
+import org.openhab.binding.rfxcom.internal.config.RFXComBridgeConfiguration;
+import org.openhab.binding.rfxcom.RFXComValueSelector;
+import org.openhab.binding.rfxcom.internal.exceptions.RFXComException;
+
+/**
+ * RFXCOM data class for control message.
+ *
+ * @author Pauli Anttila - Initial contribution
+ */
+public class RFXComInterfaceControlMessage extends RFXComBaseMessage {
+
+    private byte[] data = new byte[14];
+
+
+    public RFXComInterfaceControlMessage(RFXComInterfaceMessage.TransceiverType transceiverType, RFXComBridgeConfiguration configuration) {
+        data[0] = 0x0D;
+        data[1] = RFXComBaseMessage.PacketType.INTERFACE_CONTROL.toByte();
+        data[2] = 0;
+        data[3] = 2;
```

Comment:
Because it is in fact the second message. The first was a status request (and the zeroth was the reset). But the seqno is unimportant really. As far as the rfxcom is concerned it's just a message id that it parrots back in responses so you can match things up. Even then it isn't really useful unless you run the risk of overrunning the rfxcom's buffer (which we don't).